libr/core/cio.c

```diff
@@ -292,24 +292,12 @@ R_API bool r_core_seek(RCore *core, ut64 addr, bool rb) {
 	}
 	if (rb) {
 		ret = r_core_block_read (core);
-		if (core->io->ff) {
-			if (ret < 1 || ret > core->blocksize) {
-				memset (core->block, core->io->Oxff, core->blocksize);
-			} else {
-				memset (core->block+ret, core->io->Oxff, core->blocksize-ret);
```

Comment:
dont revert, the rebasing should be done by fixing those conflicts. as long as we are working in a squashed commit, it must be done by hand.